### PR TITLE
Add tags.

### PR DESCRIPTION
**Pull request by BeardedBear**
See if we can't improve the way things are done.

### CODE REVIEW

**Comment by n1k0** [comment]
Also there are conflicts to resolve.

**Comment by BeardedBear** [comment]
Hop  !

**Comment by n1k0** [comment]
I'm still getting this error after a git pull and bundle install:

```
➜  tech-blog git:(add-tags) bundle exec jekyll serve           
Configuration file: /home/niko/work/allomedia/tech-blog/_config.yml
            Source: /home/niko/work/allomedia/tech-blog
       Destination: /home/niko/work/allomedia/tech-blog/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
  Liquid Exception: Liquid syntax error (/home/niko/work/allomedia/tech-blog/_includes/head.html line 5): Unknown tag 'seo' included in /_layouts/default.html

```

**Comment by BeardedBear** [comment]
Can you try again @n1k0 ?

**Comment by n1k0** [comment]
Thanks, that fixed the installation part.

Now, I'm a little confused as to what do we have these entries:



Hints?

File: _posts/2018-01-26-reduce-and-the-ferris-wheel-metaphor.markdown [hunk 1]
```diff
@@ -83,10 +84,10 @@ Wow, that looks complicated, especially when you're new to Elm.
...
 In Elm, type signatures separate each function arguments and the return value with an arrow (<code style="white-space: nowrap">-></code>); so, let's decompose the one for `foldl`:
 
-- `(a -> b -> b)`, the first argument, means we want a function, taking two arguments typed `a` and `b` and returning a `b`. That sounds a lot like our `reducer` function in JavaScript! If so, `a` is an action, and `b` a state.
-- the next argument, typed as `b`, is the initial state we start reducing our list of actions from.
-- the next argument, `List a`, is our list of actions.
-- And all this must return a `b`, hence a new state. We have the exact definition of what we're after.
+* `(a -> b -> b)`, the first argument, means we want a function, taking two arguments typed `a` and `b` and returning a `b`. That sounds a lot like our `reducer` function in JavaScript! If so, `a` is an action, and `b` a state.
+* the next argument, typed as `b`, is the initial state we start reducing our list of actions from.
+* the next argument, `List a`, is our list of actions.
+* And all this must return a `b`, hence a new state. We have the exact definition of what we're after.
```
Comment (n1k0) [inline _posts/2018-01-26-reduce-and-the-ferris-wheel-metaphor.markdown]: Wow, why all these markdown changes?

File: _posts/2018-01-26-reduce-and-the-ferris-wheel-metaphor.markdown [hunk 2]
```diff
@@ -100,68 +101,68 @@ Our resulting minimalistic implementation was:
 
 {% highlight haskell %}
 type Action
-    = AddWater
-    | Empty
+= AddWater
+| Empty
```
Comment (n1k0) [inline _posts/2018-01-26-reduce-and-the-ferris-wheel-metaphor.markdown]: This breaks the initial code formatting.

File: _posts/2018-01-26-reduce-and-the-ferris-wheel-metaphor.markdown (hunk 1 above)
Comment (BeardedBear) [reply _posts/2018-01-26-reduce-and-the-ferris-wheel-metaphor.markdown]: WTF

**Pull request merged**